sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -799,3 +842,17 @@ td .formio-custom-tooltip:hover .tooltip-text {
 .grid-row-indentation-first-level {
   margin-left: 1rem;
 }
+
+/**
+* Align the container children to the left and to the right
+* to have the allowed files aligned to the left and
+* file size aligned to the right.
+*/
+.file-upload-info-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.formio-component-weight-bold > div:first-child {
+  font-weight: $font-weight-bold;
+}
```

Comment:
What is the purpose of this particular css?

Reply:
Apply a bold text to a component without interfering with other component items, for instance, error validators.

Reply:
I will add a comment.